services/assets/asset-server.mjs

```diff
@@ -0,0 +1,100 @@
+// Copyright 2021 HITCON Online Contributors
```

Comment:
There are recent changes in asset-service.mjs that was not picked up by this PR.

I would recommend not swapping asset-service and asset-server in this PR, because that can make it hard to review. As for the final state, I'm OK whether you swap asset-service and asset-server or not.